src/template.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title><%=htmlWebpackPlugin.options.title %></title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css"
+    />
+    <link rel="stylesheet" href="./mai.css" />
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <div class="header">
+          <h2>Add todo!</h2>
+        </div>
+        <div class="new-todo">
+          <form id="todoform">
+            <input
+              type="text"
+              name="newtodo"
+              id="newtodo"
+              placeholder="I will do ..."
+            />
+            <button type="submit">
+              <i class="bi bi-plus-circle-fill"></i>
+            </button>
+          </form>
+        </div>
+      </header>
+      <div id="todos-list todo</div>
```

Comment:
- [ ] there is a runtime error because you forgot to close the quote in this div id  it should be like this ` <div id="todos-list "> todo</div>` kindly fix this
 
![Screenshot from 2023-05-11 16-01-14](https://github.com/husush23/todo-list/assets/66097371/2c653ea6-f99d-489d-9203-f22ab981c20a)